_api-reference/profile.md

```diff
@@ -756,274 +989,11 @@ Field | Description
-
-#### The `collector` array
-
-|Field	|Description	|
-|:---	|:---	|
-|`time_in_nanos`	|The total elapsed time for this collector, in nanoseconds. For concurrent segment search, `time_in_nanos` is the total amount of time across all slices (the difference between the last completed slice execution end time and the first slice execution start time).	|
-|`max_slice_time_in_nanos`	|The maximum amount of time taken by any slice, in nanoseconds.	|
-|`min_slice_time_in_nanos`	|The minimum amount of time taken by any slice, in nanoseconds.	|
-|`avg_slice_time_in_nanos`	|The average amount of time taken by any slice, in nanoseconds.	|
-|`slice_count`	|The total slice count for this query.	|
-|`reduce_time_in_nanos`	|The amount of time taken to reduce results for all slice collectors, in nanoseconds.	|
-
-#### The `aggregations` array
-
-|Field	|Description	|
-|:---	|:---	|
-|`time_in_nanos`	|The total elapsed time for this aggregation, in nanoseconds. For concurrent segment search, `time_in_nanos` is the total amount of time across all slices (the difference between the last completed slice execution end time and the first slice execution start time).	|
-|`max_slice_time_in_nanos`	|The maximum amount of time taken by any slice to run an aggregation, in nanoseconds.	|
-|`min_slice_time_in_nanos`	|The minimum amount of time taken by any slice to run an aggregation, in nanoseconds.	|
-|`avg_slice_time_in_nanos`	|The average amount of time taken by any slice to run an aggregation, in nanoseconds.	|
-|`<method>`	|The total elapsed time across all slices (the difference between the last completed slice execution end time and the first slice execution start time). For example, for the `collect` method, it is the total time spent collecting documents into buckets across all slices.	|
-|`max_<method>`	|The maximum amount of time taken by any slice to run an aggregation method.	|
-|`min_<method>`|The minimum amount of time taken by any slice to run an aggregation method.	|
-|`avg_<method>`	|The average amount of time taken by any slice to run an aggregation method.	|
-|`<method>_count`	|The total method count across all slices. For example, for the `collect` method, it is the total number of invocations of this method needed to collect documents into buckets across all slices.	|
-|`max_<method>_count`	|The maximum number of invocations of a `<method>` on any slice.	|
-|`min_<method>_count`	|The minimum number of invocations of a `<method>` on any slice.	|
-|`avg_<method>_count`	|The average number of invocations of a `<method>` on any slice.	|
+`<method>`	|The total elapsed time across all slices (the difference between the last completed slice execution end time and the first slice execution start time). For example, for the `collect` method, it is the total time spent collecting documents into buckets across all slices.	
```

Comment:
This aggregation "method" field here also means the aggregation breakdown field like "initialize", "build_leaf_collector", ..., "reduce". So we can remove this row, and add the concurrent segment search description above.

Reply:
Basically, we can remove this row.